omise/setting.php

```diff
@@ -0,0 +1,86 @@
+<?php
+if (! defined('_PS_VERSION_')) {
+    exit();
+}
+
+class Setting extends PaymentModule
+{
+    protected $submit_action = 'omise_save_setting';
+
+    public function getLivePublicKey()
+    {
+        return Configuration::get('live_public_key');
+    }
+
+    public function getLiveSecretKey()
+    {
+        return Configuration::get('live_secret_key');
+    }
+
+    public function getPublicKey()
+    {
+        if ($this->isSandboxEnabled()) {
+            return Configuration::get('test_public_key');
+        }
+
+        return Configuration::get('live_public_key');
```

Comment:
@guzzilar It has been improved in the next pull request. So, it is not necessary to comment here.